MapboxNavigation/ErrorCodes.swift

```diff
@@ -0,0 +1,36 @@
+import Foundation
+
+/**
+ Enum used for indicating the type of error occured while speaking an instruction.
+ */
+public enum SpokenVoiceError: Int {
```

Comment:
There should be a single enumeration with all the error codes we may need to use in this library, similar to the map SDK’s `MGLErrorCode`.